pkg/riff/commands/stream_create.go

```diff
@@ -77,7 +78,7 @@ func NewStreamCreateCommand(c *cli.Config) *cobra.Command {
 	}
 
 	cli.NamespaceFlag(cmd, c, &opts.Namespace)
-	cmd.Flags().StringVar(&opts.Provider, cli.StripDash(cli.ProviderFlagName), "", "<todo>")
+	cmd.Flags().StringVar(&opts.Provider, cli.StripDash(cli.ProviderFlagName), "", "stream provider")
```

Comment:
Needs to explain what a provider is. Maybe that should go in the command description.